pandevice/network.py

```diff
@@ -1196,7 +1202,22 @@ def _setup(self):
             'dhcp_default_route_metric',
             path='{mode}/dhcp-client/default-route-metric',
             vartype='int', condition={'mode': 'layer3'}))
-
+        params.append(VersionedParamPath(
+            'lacp_enable',
+            condition={'mode': ['layer3', 'layer2', 'ha']},
+            vartype='yesno', path='{mode}/lacp/enable'))
+        params.append(VersionedParamPath(
+            'lacp_passive_pre_negotiation',
+            condition={'mode': ['layer3', 'layer2', 'ha'], 'lacp_enable': True},
+            vartype='yesno', path='{mode}/lacp/passive-pre-negotiation'))
+        params.append(VersionedParamPath(
+            'lacp_mode',
+            condition={'mode': ['layer3', 'layer2', 'ha'], 'lacp_enable': True},
+            path='{mode}/lacp/mode'))
+        params.append(VersionedParamPath(
+            'lacp_rate',
+            condition={'mode': ['layer3', 'layer2', 'ha'], 'lacp_enable': True},
+            path='{mode}/lacp/transmission-rate'))
```

Comment:
The docstring mentions valid values for these two.  Add a `choices=[...]` to these two to assist users.

Reply:
Assuming you meant values not choices, right?  I don't see choices in VersionedParamPath or ParamPath class.